.gitignore

```diff
@@ -22,3 +22,9 @@
 *.gz*
 *notes*
 .DS_Store*
+
+# GoLand files
```

Comment:
Would rather this arrives in a separate PR, but it's fine we can merge it in here 😸

Reply:
Heh, okay, thanks!